webapp/src/Service/ImportExportService.php

```diff
@@ -703,15 +703,15 @@ public function importGroupsJson(array $data, ?string &$message = null, ?array &
             $groupData[] = [
                 'categoryid' => @$group['id'],
                 'icpc_id' => @$group['icpc_id'],
-                'name' => @$group['name'],
+                'name' => $group['name'] ?? '',
```

Comment:
Wouldn't all properties need to be optional/nullable in this case?

Reply:
Nom name is required, and can't be null since the method doesn't accept null. The other things do